packages/twenty-server/src/engine/core-modules/function/function.resolver.ts

```diff
@@ -0,0 +1,38 @@
+import { UseGuards } from '@nestjs/common';
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+
+import { FileUpload, GraphQLUpload } from 'graphql-upload';
+
+import { JwtAuthGuard } from 'src/engine/guards/jwt.auth.guard';
+import { AuthUser } from 'src/engine/decorators/auth/auth-user.decorator';
+import { User } from 'src/engine/core-modules/user/user.entity';
+import { FunctionService } from 'src/engine/core-modules/function/function.service';
+import { ExecuteFunctionInput } from 'src/engine/core-modules/function/dtos/execute-function.input';
+
+@UseGuards(JwtAuthGuard)
+@Resolver(() => String)
+export class FunctionResolver {
+  constructor(private readonly functionService: FunctionService) {}
+
+  @Mutation(() => String)
+  async upsertFunction(
```

Comment:
we should align on other metadata services (that's being said we want to deprecate nestjs-query-graphql)